package.json

```
@@ -54,4 +58,4 @@
     "node": "^4.5 || 6.* || >= 8.*"
```

We should probably also bump this to:

```suggestion
    "node": "8.* || 10.* || >= 12.*"
```